samples/bluetooth/radio_coex_3wire/src/main.c

```diff
@@ -0,0 +1,297 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stddef.h>
+#include <sys/printk.h>
+#include <console/console.h>
+#include <sys/__assert.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/uuid.h>
+
+#include <hal/nrf_radio.h>
+#include <helpers/nrfx_gppi.h>
+#include <nrfx_timer.h>
+#include <nrfx_gpiote.h>
+#include <hal/nrf_egu.h>
+#include <nrfx_ppi.h>
+
+#define ZEPHYR_USER_NODE DT_PATH(zephyr_user)
+#if (DT_GPIO_PIN(ZEPHYR_USER_NODE, coex_pta_req_gpios) && \
+	DT_GPIO_PIN(ZEPHYR_USER_NODE, coex_pta_grant_gpios) && \
```

Comment:
Nice to see this using `/zephyr,user`. However, this test should be using DT_NODE_HAS_PROP, not DT_GPIO_PIN.

Reply:
doesn't work for some reason, maybe phandles are treated differently ? I tried to check if we had anything phandle-specific, but couldn't find any

Reply:
alright, I must've had some bad syntax before because it works now, cheers !